doc/ref_arch/kubernetes/chapters/appendix-b.md

```diff
@@ -0,0 +1,43 @@
+[<< Back](../../kubernetes)
+# Appendix B - Guidance For workload isolation(Multitenancy) with Kubernetes for application Vendors
+<p align="right"><img src="../figures/bogo_sdc.png" alt="scope" title="Scope" width="35%"/></p>
+
+## Table of Contents
+
+* [B.1 Overview](#B.1)
+* [B.2 Multitenancy Models for CNFs](#B.2)
+* [B.3 Solution Areas](#B.3)
+
+
+<a name="B.1"></a>
+## B.1 Overview
+
+Problem statement: A single Kubernetes cluster does not provide hard multitenancy by design. Within a Cluster, Namespace is a mechanism to provide Soft isolation multitenancy.
```

Comment:
```suggestion
Problem statement: A single Kubernetes cluster does not provide hard multitenancy by design. Within a Cluster, Kubernetes Namespace is a mechanism to provide Soft isolation multitenancy.
```

We should somehow differentiate between kernel namespaces and Kubernetes namespaces.